src/main/groovy/io/openapiprocessor/gradle/OpenApiProcessorTask.java

```diff
@@ -35,6 +37,14 @@ abstract public class OpenApiProcessorTask extends DefaultTask {
     @InputDirectory
     abstract public Property<String> getApiDir();
 
+    /**
+     * Any other files used by the processor that are considered inputs. Used by gradle for the up-to-date check.
+     *
+     * @return the input files
+     */
+    @InputFiles
+    abstract public ListProperty<String> getInputFiles();
```

Comment:
What do you think about this approach @hauner? It didn't feel 100% to hardcode this as "mapping", but if you think that would make it more clear I don't mind changing it.

I tested this locally against my own project and Gradle now correctly triggers processing whenever the mapping file is changed.

Reply:
What would be the use case for having many files here? 

Currently, if the `mapping.yaml` is a sibling to `openapi.yaml`, gradle recognizes a change to it  by the  `@InputDirectory apiDir` (i.e. its parent directory).

The new `@Input...` would help to recognize changes to `mapping.yaml` if we put it somewhere else.  

I think it would make more sense to explicitly name this `mapping`, a single file `@InputFile` would be enough for that.

Hopefully I remember that correctly, didn't look at this for a while. :-)